app/src/features/projects/view/ViewProjectPage.tsx

```diff
@@ -252,16 +233,6 @@ const ViewProjectPage: React.FC = () => {
                   <Paper elevation={2}>
                     <Box height="500px" position="relative">
                       <LocationBoundary locationData={project.location} />
```

Comment:
Are we not supporting fullscreen ever? I am unsure of the requirements

Reply:
We could... It would be a separate Maplibre plugin though.

Reply:
Sounds good!

Reply:
https://maplibre.org/maplibre-gl-js/docs/examples/fullscreen/